src/zkevm_specs/util/ec.py

```diff
@@ -0,0 +1,113 @@
+from typing import Tuple
+from .arithmetic import FQ
+from eth_keys import KeyAPI  # type: ignore
+
+
+class WrongFieldInteger:
```

Comment:
Personally, prefer to move to a single file since not only place is using. I was trying not to touch unrelated code in a single PR so I didn't merge those two into one. I forgot to create a issue to clean up duplicated code. I can clean this up in this PR since it won't take too much work. What is your opinion?

Reply:
Got it, make sense, we could refactor that in further PR